### PR TITLE
More reorder and cleanups

### PR DESCRIPTION
**Pull request by fgsch**
Hopefully last changes as part of #808.

### CODE REVIEW

**Comment by fgsch** [comment]
Do not merge yet. 

I'm working my way through REQUEST-942-APPLICATION-ATTACK-SQLI.conf.

**Comment by dune73** [comment]
You are my hero, @fgsch.

One question: You are removing some of the line breaks. This makes our lines overly long. Maybe more readable, but you need to scroll.

I'd rather keep the line break, though. Sorry if you talked this through and I missed that. But I am inclined to bring it back to the table. Maybe we should add a remark to the contributing document.

**Comment by fgsch** [comment]
I modified those 3 entries because all but those rules do not have breaks after the variable but I'm happy to revert this part and change all the existing ones, though considering some (most?) of the operators are rather long this will not make a big difference.

IOW, for consistency I prefer to break these 3 rules than changing the rest but happy to do either way.

**Comment by dune73** [comment]
Yes, it seems as if these line breaks on exactly these three rules would be exceptions and completely arbitrary.

Can we have some additional opinions here please? @csanders-git, @lifeforms? Anybody else?

**Comment by fzipi** [comment]
I am ok with these changes, and I too prefer consistency.

**Comment by lifeforms** [comment]
I would prefer keeping the line breaks to prevent wrapping, but I have no very strong opinion.

**Comment by fgsch** [comment]
Done with all the updates to this PR. 

Ready to be merged but please take a look and let me know.

**Comment by fzipi** [comment]
For me is good to go!

**Comment by dune73** [comment]
Let me take a look. Did not have the time so far, but I'll be back.

**Comment by dune73** [comment]
That looks good. Merging now.

Just as a note: This PR removes the only three line breaks after the SecRule variable in the rule set. This means all the rules come with SecRules, variable, operator, parameter and then linebreak now.

This might not be the best way to do it, but it is consistent now and can be discussed elsewhere.

Thank you for the PR @fgsch.

**Comment by emphazer** [comment]
@fgsch i get an error now.
`````
Syntax error on line 105 of /etc/httpd/modsecurity.d/activated_rules/REQUEST-910-IP-REPUTATION.conf:
Error creating rule: Could not open ipmatch file "/etc/httpd/modsecurity.d/activated_rules/ip_blacklist.data": No such file or directory
`````
rule 910110 was disabled by default.
why did you remove the ````#```` of rule 910110 ?
maybe an accident?

File: rules/REQUEST-910-IP-REPUTATION.conf [hunk 1]
```diff
@@ -89,23 +89,23 @@ SecRule TX:HIGH_RISK_COUNTRY_CODES "!@rx ^$" \
 # This rule checks the client IP address against a list of recent IPs captured
 # from the SpiderLabs web honeypot systems (last 48 hours).
 #
-#SecRule TX:REAL_IP "@ipMatchFromFile ip_blacklist.data" \
```
Comment (emphazer) [inline rules/REQUEST-910-IP-REPUTATION.conf:92]: @fgsch here

Comment (fgsch) [reply rules/REQUEST-910-IP-REPUTATION.conf:92]: Oops!

Yes, looks like I was overly zealous when going through it. Sorry.

I can open a PR tonight/tomorrow to add the comment back, or happy to approve it if you raise one before me.

Comment (dune73) [reply rules/REQUEST-910-IP-REPUTATION.conf:92]: Sorry. Should have caught this too. Sorry.

I assumed @fzipi had checked the correct working and I only looked through the files without actually loading them.

Negligence like this is biting us time and time again. We need travis to launch server and load the rules during the testing.

Personally, I do not have time for a PR to fix the issue today.

Comment (fgsch) [reply rules/REQUEST-910-IP-REPUTATION.conf:92]: Mistake aside, in all fairness I don't think we should include commented out rules.
AFAICT this is the only occurrence.

Comment (fgsch) [reply rules/REQUEST-910-IP-REPUTATION.conf:92]: Opened PR https://github.com/SpiderLabs/owasp-modsecurity-crs/pull/973 to correct this.

Comment (fzipi) [reply rules/REQUEST-910-IP-REPUTATION.conf:92]: @dune73 We need to launch a travis test for this one, agreed. I've been ocassionally using ```modsec-rules-check``` from ModSecurity v3 for this also. I've also read it, but from the reordering perspective. It happened to me also.

**Pull request merged**